example/metro.config.js

```diff
@@ -14,18 +14,7 @@ const modules = Object.keys({...pak.peerDependencies});
  * @type {import('metro-config').MetroConfig}
  */
 const config = {
-  watchFolders: [root],
```

Comment:
Explanation in https://github.com/Expensify/react-native-live-markdown/pull/529/files#r1819209115